Automattic/vip-design-system
# Update the components colors based on the latest Valet Design System

## Description

Ps: *Dark mode is not part of the scope*

<img width="2728" alt="image" src="https://user-images.githubusercontent.com/3402/208719572-c68b865c-5ced-4193-a7b1-70f8fb40154f.png">

This PR uses the latest develop/tokens/* files and adapts the components using the new color system. All possible components are being updated.

Preview URL: https://deploy-preview-138--vip-design-system-components.netlify.app/

All changes were made in sync with @donaghokeeffe (VIP Designer), matching the latest Figma files from the Valet Design System.

# Steps to test

1. Go to https://deploy-preview-138--vip-design-system-components.netlify.app/
2. Navigate into all non-deprecated components pages
3. Colors have changed. You should not find any accessibility contrast issues 

<img width="939" alt="image" src="https://user-images.githubusercontent.com/3402/209171090-5b4cd1b0-2552-4ce0-aecb-4246adb12483.png">

# Steps to test (Optional)

1. Go to the vip-dashboard repo
2. Change the `"@automattic/vip-design-system": "0.27.10",` to `"@automattic/vip-design-system": "github:automattic/vip-design-system#f870605e8312c3b90bd4d5981fc8a1f0c85cc98d"`
3. Do `npm install`
4. Do `npm run dev`
5. Test the dashboard works as expected with the new colors
